src/link/assign.c

```diff
@@ -42,7 +67,7 @@ area_Avail(SLONG bank)
 	return (r);
 }
 
-SLONG 
+bool
```

Comment:
Well, just a 0 for success and 1 or -1 for error would be ok. We don't really need `org`, I guess...